modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuildController.java

```diff
@@ -368,21 +381,22 @@ private void tryScheduleBuildIndex(
             ReplicationGroupId primaryReplicaId,
             CatalogIndexDescriptor indexDescriptor,
             MvTableStorage mvTableStorage,
-            ReplicaMeta replicaMeta
+            ReplicaMeta replicaMeta,
+            HybridTimestamp buildAttemptTimestamp
     ) {
         // TODO https://issues.apache.org/jira/browse/IGNITE-22522
         // Remove TablePartitionId check.
         assert primaryReplicaId instanceof ZonePartitionId
                 ? ((ZonePartitionId) primaryReplicaId).zoneId() == zoneId
-                        && ((ZonePartitionId) primaryReplicaId).partitionId() == partitionId
+                && ((ZonePartitionId) primaryReplicaId).partitionId() == partitionId
```

Comment:
Here, the line is outdented, but a similar line (2 lines below) is not. It looks like an inconsistency

Reply:
Fixed.